pkg/operator/status.go

```diff
@@ -309,6 +311,30 @@ func (c *Controller) syncStatus(cr *imageregistryv1.Config, deploy *appsapi.Depl
 	} else if cr.Spec.ManagementState == operatorapiv1.Removed {
 		operatorDegraded.Message = "The registry is removed"
 		operatorDegraded.Reason = "Removed"
+	} else if deploy == nil {
+		operatorDegraded.Status = operatorapiv1.ConditionTrue
+		operatorDegraded.Message = "The deployment does not exist"
+		operatorDegraded.Reason = "DeploymentNotFound"
```

Comment:
If we shouldn't immediately react on absence of the deployment. Most likely the operator will fix this problem in a second.

Reply:
Yes, you are absolutely right. I have adjusted it by means of removing it, now it will take one minute as well. PTAL.